igvc/src/speed_compare/main.cpp

```diff
@@ -0,0 +1,77 @@
+#include <igvc_msgs/velocity_pair.h>
+#include <ros/ros.h>
+#include <std_msgs/Bool.h>
+#include <chrono>
+#include <fstream>
+
+bool enabled = false;
+std::ofstream file;
+float target_left_vel;
+float target_right_vel;
+ros::Time start_time;
+
+void enabled_callback(const std_msgs::BoolConstPtr& msg)
```

Comment:
Just a note here. On the actual robot /robot_enabled will be continually published (10Hz I think) so this will stream output. I do not think that is a problem just a note here.